installer/templates/new/web/web.ex

```diff
@@ -48,9 +48,11 @@ defmodule <%= application_module %>.Web do
     end
   end
 
-  def view do
+  def view(opts \\ []) do
```

Comment:
Let's please drop the changes to this file. You are welcome to do those changes in your application but I don't think they belong to all apps by default.